.github/workflows/pull-request.yaml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         include:
-          - rust: 1.70.0
+          - rust: 1.77.2
```

Comment:
According to the readme the crate has a minimum supported rust version of 1.64 but I guess that ship has already sailed. 😅 
Is there a reason this needed to be updated? BTW this exact version is already covered by `stable` 3 lines down.

Reply:
This update is required by the AWS stable release, otherwise it won't compile.

Reply:
Overall I'm not a fan of bumping the rust version here. The version you are adding only requires 1.75 according to their readme (MSRV = stable - 2).
Also with the discussion on compatibility of sub-dependency version I think we should not specify the patch level version in our dependencies to give the dependency resolver more room to unify dependency versions.
Also since the MSRV definitely has to be increased (I agree that depending on a stable version of the aws crates makes sense) we should also update the readme of this repo since it mention MSRV as well.
And finally this needs to get published as a new version of the crate. Since we are increasing the MSRV I think to make it a new minor release instead of just doing a patch level release (i.e. 0.25.6 -> 0.26.0). This can be done by using `/ci/bump.sh`.